.circleci/config.yml

```diff
@@ -145,7 +145,7 @@ jobs:
   test-gyp:
     <<: *test-base-legacy
     docker:
-      - image: gcc:4.8.5
+      - image: gcc:7
```

Comment:
GCC 4 image appears to no longer allow successful clone of project. Given GYP test is mostly for Protagonist, I am not concerned if GCC 4 is no longer supported, given that no modern supported version of NodeJS support GCC 4.